parsers/uottawa.js

```diff
@@ -37,7 +38,7 @@ function parse(text) {
       var professor = _professorRegex.exec(classes[j]);
       var time = _timeRegex.exec(classes[j]);
       var where = _locationRegex.exec(classes[j]);
-      if (!professor || !time || !where) continue;
```

Comment:
Nope, it's left empty because they haven't been assigned yet. Usually happens 2-3 weeks before semester start.